packages/devextreme/js/__internal/grids/grid_core/row_dragging/m_row_dragging.ts

```diff
@@ -213,6 +217,7 @@ const rowsView = (Base: ModuleType<RowsView>) => class RowsViewRowDraggingExtend
         fixed: column.fixed,
         fixedPosition: column.fixedPosition,
       })),
+      isDragging,
```

Comment:
Is it possible to just write: `isDragging: true`? I suggest to do that because this options is passed only to draggable grid, so it should be always true.